package.json

```diff
@@ -1,23 +1,28 @@
 {
   "name": "pci-dss-sanitizer",
-  "version": "0.0.0-development",
+  "version": "v0.1.0",
```

Comment:
Well, yes intentionally because there was no version yet?

Reply:
No the idea is that you keep it locally to `0.0.0-development` and semantic release will compare current changes to new changes and figure out the new version number.

Reply:
Yes but it doesn't update package.json, it only creates a new release.

Reply:
That's the point :p so that you as a dev don't have to worry about it and fix possible merge conflicts in package.json. Also, now when we want to create a release we should manually check the commits, figure out what the next version number will be and update the correct version number and release it. Note that we are not in control of the end version it is computed by commit messages. This means if we think it is 0.2.1 and semantic release creates a 0.3.0 release then we will have a hard time I think...

Reply:
Hm, true. 🤔 😄